CMakeLists.txt

```diff
@@ -301,3 +309,29 @@ endif()
 if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
   set(CMAKE_INSTALL_LIBDIR lib)
 endif()
+
+install(
+    TARGETS azure-storage-lite
+    EXPORT azure-storage-lite-export
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+)
+install(
```

Comment:
Are all header files in include should be treated as public headers? If not, propose to get public header list and install  similar to this:

`set_target_properties(azure-storage-lite PROPERTIES PUBLIC_HEADER ${headers})`
`install(TARGETS azure-storage-lite PUBLIC_HEADER)`